libs/partners/google-genai/langchain_google_genai/llms.py

```diff
@@ -259,6 +259,11 @@ def _stream(
     def _llm_type(self) -> str:
         """Return type of llm."""
         return "google_palm"
+    
+    @property
```

Comment:
nits: maybe it's time to introduce `BaseGoogleGenerativeAI` class and avoid code duplication between chat and llm models, wdyt?

Reply:
Fair point. On it

Reply:
Done